aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -315,5 +315,32 @@ Tensor& matmul_out(Tensor &result, const Tensor & tensor1, const Tensor & tensor
   return result;
 }
 
+Tensor frobenius_norm(const Tensor& self) {
+  IntList dim = {0, 1};
+  return at::native::frobenius_norm(self, dim);
 }
+
+Tensor frobenius_norm(const Tensor& self, IntList dim, bool keepdim) {
+  AT_CHECK(
+      dim.size() <= 2,
+      "Improper number of chosend dimensions to do frobenius norm.");
+  if (dim.size() == 0) {
```

Comment:
Why would this case arise?

Reply:
Dim are the dimensions will be reduced. You could not reduce more than two dimensions when calculating Frobenius Norm.

Reply:
I am guessing the parsing happens like this:
```
frobenius_norm(torch.Tensor, python list, python bool)
```
Why would someone pass an empty list?